.github/workflows/post-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Post merge
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    uses: .github/workflows/run-tests.yml@master
```

Comment:
If it's a file in-repo you don't use the @master, that's just for checking out other repos. You also need to start the path with `./` to indicate it's local to the repo:

https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow